src/V3FactoryOwner.sol

```diff
@@ -169,6 +169,14 @@ contract V3FactoryOwner {
   /// amount are. Effectively, as each pool accrues fees, it eventually becomes possible to "buy"
   /// the pool fees for less than they are valued by "paying" the the payout amount of the payout
   /// token.
+  /// `payoutAmount` may be changed by the admin (governance). Any proposal that changes this amount
+  /// is expected to be subject to the governance process, including a timelocked execution, and so
+  /// it's unlikely that a caller would be surprised by a change in this value. Still, callers
+  /// should be aware of the edge case where:
+  /// - 0. The caller grants a higher-than-necessary payout token approval to this factory.
+  /// - 1. caller's claimFee transaction is in the mempool.
+  /// - 2. the payoutAmount is changed.
+  /// - 3. the claimFee transaction is now included in a block.
```

Comment:
It would look cleaner if we used either bullets or numbering rather than both

Reply:
Docs look good, agree with this nit from @alexkeating, but otherwise good to go!

Reply:
good call, fixed in 5b76a0d